### ISSUE TITLE
Label of train set

### ISSUE THREAD

**Issue by pynojs** [official-stockfish/nnue-pytorch#288]
May I ask how the labels in the training set, i.e. the outputs, are obtained?
